src/components/search/index.tsx

```diff
@@ -76,38 +81,27 @@ const Search: FunctionComponent<SearchProps> = ({
     ? `Search resources by ${instructor.full_name}`
     : undefined
 
-  const shouldDisplayLandingPageForTopics = (topics: string | string[]) => {
-    topics = isArray(topics) ? topics : [topics]
+  const shouldDisplayLandingPageForTopics = (topic: string) => {
     return (
-      (searchState?.query &&
-        !isEmpty(topics) &&
-        isEmpty(searchState.page) &&
-        topics.includes(searchState?.query?.toLowerCase()) &&
-        numberOfRefinements === 0) ||
-      (isEmpty(searchState.query) &&
-        isEmpty(searchState.page) &&
-        numberOfRefinements === topics.length &&
-        noInstructorsSelected(searchState) &&
-        onlyTheseTagsSelected(topics, searchState))
+      isEmpty(searchState.query) &&
+      isEmpty(searchState.page) &&
+      noInstructorsSelected(searchState)
     )
   }
 
-  const CURATED_PAGES = ['react', 'javascript', 'graphql']
-
-  const shouldDisplayDefault = (topics: string | string[]) => {
-    topics = isArray(topics) ? topics : [topics]
-    const tags = get(searchState, 'refinementList._tags', [])
-
-    const tag = tags?.[0]
-
-    return tag && numberOfRefinements === 1 && !topics.includes(tag)
-  }
-
   const shouldDisplayLandingPageForInstructor = (slug: string) => {
-    return typeof InstructorsIndex[slug] !== 'undefined'
+    return (
+      isEmpty(searchState.query) &&
+      isEmpty(searchState.page) &&
+      noTopicsSelected(searchState)
+    )
   }
 
-  const InstructorCuratedPage = instructor && InstructorsIndex[instructor.slug]
+  const InstructorCuratedPage =
+    instructor &&
+    (InstructorsIndex[instructor.slug] || SearchInstructorEssential)
```

Comment:
if we have a curated page, use that, if not we have a sensible default that works!